atom/browser/api/atom_api_window.h

```diff
@@ -54,6 +54,10 @@ class Window : public mate::TrackableObject<Window>,
 
   int32_t ID() const;
 
+  static void AddGlobalPreload(const base::FilePath::StringType preloadPath);
+  static void RemoveGlobalPreload(const base::FilePath::StringType preloadPath);
```

Comment:
Could this be `const base::FilePath::StringType&` to avoid a copy?

Reply:
Done 👍